prov/terraform/commons/resources.go

```diff
@@ -25,6 +25,8 @@ const (
 	// DefaultSSHPrivateKeyFilePath is the default SSH private Key file path
 	// used to connect to provisioned resources
 	DefaultSSHPrivateKeyFilePath = "~/.ssh/yorc.pem"
+	// NullPluginVersionConstraint is the Terraform null plugin version constraint
+	NullPluginVersionConstraint = "~> 1.0"
```

Comment:
Why this is defined here while others plugin are in config?

Reply:
I guess this plugin is not supposed to change often.

Reply:
ok